canonical-web-and-design/snapcraft.io
# Ensure developer isn't cached outside publisher pages

## Done
- Copy and pasted from charmhub

## How to QA
- Visit the demo
- Login
- Go back to the homepage - you should see your fullname in the header

## Issue / Card
Fixes canonical-web-and-design/marketplace-tribe/2057
